internal/api/server/server.go

```diff
@@ -19,7 +20,12 @@ func RunServer(initiator common.ServerInitiator) (err error) {
 		return err
 	}
 
-	core.SetAPIDescription(*api.Describe(*api.GetRoutes()))
+	apiDescription, errs := api.Describe(*api.GetRoutes())
+	if errs != nil {
+		log.Warn("Found errors when describing the API endpints.", errs)
+	}
```

Comment:
What errors are you thinking of that we could encounter, but still run the server?

I know an error in the description may not be something that actually stops the server from running, but we could force it to stop so people resolve the issue.